GTBitsOfGood/canine-assistants
# Daniel: User management dashboard

## [FE] User management page PR

Issue Number(s): #84

What does this PR change and why?
This PR implements the entire user management dashboard (display name and email, change user role, toggle active/inactive state)

### Checklist

### Critical Changes
- Database change / migration to run: User model may need to be tweaked in the future
- Environment config change
- Breaking API change

### Related PRs

PR Number(s): #number_of_pr

### How To Test

Search functionality- type user names into search bar and confirm it is correct
Dropdown boxes- update the user roles via the dropdown, ensure that they are actually updated in the database and reflected through the UI
Toggle Switch- Toggle switch off to confirm inactive role is set. User can be activated by setting a "Admin" or "User" role through the dropdown box
